source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -731,6 +777,7 @@ export default class WalletSendForm extends Component<Props, State> {
                 <button
                   onClick={() => this.handleOnReset()}
                   className={styles.clearReceiverButton}
+                  tabIndex={-1}
```

Comment:
@aleksandardjordjeviciohk why do we need this? What is it for?

Reply:
There was an issue with tab indexing and this button was taking focus out of tab focus on input fields, so we are now skipping it.